.huskyrc

```diff
@@ -0,0 +1,6 @@
+{
+  "hooks": {
+    "pre-commit": "lint-staged",
+    "pre-push": "yarn test:all"
```

Comment:
Maybe we should leave this to the pipeline? It takes painfully long to run test for all files in every push :/

Reply:
:) then the whole PR is useless. Yes, it takes time, maybe couple of minutes more.. but then its just double check, that repo always has the "right" and working version. Pipeline can double check of course.. because its not a 100% way to restrict pushing the bad code.

Reply:
I mean we could run the tests pre-commit for staged files